umake/tools.py

```diff
@@ -447,13 +448,16 @@ def add_env_to_user(framework_tag, env_dict):
             os.environ[env] = value
         envs_to_insert[env] = value
 
+    if env.startswith("alias "):
+        profile_filepath = profile_filepath.replace("profile", "bashrc")
+
     with open(profile_filepath, "a", encoding='utf-8') as f:
         f.write(profile_tag.format(framework_tag))
         for env in envs_to_insert:
             value = envs_to_insert[env]
             logger.debug("Adding {} to user's {} for {}".format(value, env, framework_tag))
             export = ""
-            if env != "PATH":
+            if env != "PATH" and not env.startswith("alias "):
```

Comment:
Should we have a test for this case?

Reply:
We have now a simple test in test_tools.py